### DISCUSSION TITLE
ARCHIVIST - researchers using data

### DISCUSSION THREAD

**Discussion by birk** [pair-cg/general-contributions#33] [User Stories]
# 1. Need:
As an archivist I want to provide performing arts researchers with data that supports them in answering their research questions.

# 2. Action:
* Create lots of data on the holdings of my archive.
* Try to be detailed when doing so.
* Possibly collect data on performing arts in my country in general.
* Connect all the data to provide multiple viewpoints that are not limited to the archival provenance precept. 
* Publish the data in a somewhat open and accessible way.

# 3. Desired Outcome:
* Researchers find my data.
* They can trust the data.
* They can combine it with data they received from other sources.
* They are able to draw their own conclusions from the data.
* Optionally, the researchers have combined my data with the data they have collected and donate their data to my archive.

# 4. Perspective:
Producer of information - and maybe also consumer of the information researchers have gathered.

# 5. Stakeholder Group:
* ARCHIVIST
* RESEARCHER

# 6. Additional info:
Of course, I have an idea what kind of data researchers might need. But I do not want to limit the information or format to specific research questions or practices.

**Comment by nprobst1** [comment]
This is the user story I am most interested in (as a researcher working in an archive). The data would not only have to be presented as a data base but prefereably as raw data/an API.

**Comment by drosu** [comment]
Hi Birk, 
(1) Would the following "As an archivist I want to ensure that the data/information I manage conforms to the FAIR Data Principles", i.e., findability, accessibility, interoperability, and reusability (https://en.wikipedia.org/wiki/FAIR_data) be an accurate description of the need central to this user story? 
(2) Based on the reference to data trustworthiness in the "Desired Outcomes", I suggest adding a new user story centered around the need to assist information seekers in assessing the trustworthiness of the data/information they access (data/information under the management/custody of the archivist). (People accessing the data should be able to conduct their own assessment, but they would need access to (meta)information that would support that process, including perhaps a description of the processes used by the archive to authenticate and evaluate the information it manages.)
(3) Can you give more details related to the desired outcome of allowing researchers to draw their own conclusions - is it related to collecting and making available data/information that is unaggregated, minimally biased, etc.
(4) I suggest adding a new user story related to the desired outcome of receiving the results of research conducted using data belonging to the archive, and integrating it with the existing information. Perhaps this user story could elaborate a bit on what kind of data/information the archive is looking to receive? - This user story could be part of a larger group of user stories related to the acquisition of data/information by archives, i.e., the archivist (the archive they manage) is the consumer rather than the producer of content.
Thank you!

**Reply by nprobst1** [reply]
I would guess that the FAIR-principles are implied. Also, I think we talked about the importance to consider the CARE-prinicples as well (https://www.gida-global.org/care) - Collective Benefit, Authority to Control, Responsibility, and Ethics.

**Reply by birk** [reply]
I just wasn't sure wether describing the data as FAIR is already too much for a user story. FAIR is not part of my problem, it is part of the solution. ;) It is not FAIR data that I want, I want findable data.

**Reply by drosu** [reply]
(1) A clarification is in order :) My question was not meant to be  understood as an endorsement of the FAIR principles or as a suggestion for replacing the original formulation with one that  references the FAIR principles. I wanted to validate my interpretation  of the original need formulation, i.e., "provide performing arts researchers with data that supports them in answering their research  questions" and of the desired outcomes. My interpretation of this user  story was that the archivist needs to ensure that the data/information  under their custody in (in a general sense): (a) discoverable and/or findable, (b) accessible,  (c) interoperable, (d) reusable and (e) relevant to the needs of potential  users of the archive (e.g., researchers). If my interpretation is at least  partially correct, I would like to start a discussion about rephrasing the  original need and perhaps splitting this user story into multiple user story  “typologies” that can, in turn, be refined to a level of granularity that  would make it possible for PAIR-CG to  design and/or extract relevant  terminology / concepts / definitions / axioms. It would be really helpful  to see concrete examples of how making certain archival data/artifacts  available helped with a research project. 
As a side note, (a), (b), (c)  and (d) have a technology component (APIs for directly querying the data  or for placing a request that needs to be manually processed, making data  available in standard data exchange formats, e.g., JSON, XML, RDF, etc.)  as well as an knowledge representation component (e.g., physical artifacts
in the archive have associated digital descriptions that contain keywords  from information representation frameworks that performing arts professionals  and researchers are familiar with and likely to use).
(2) Without endorsing the CARE-principles, I would welcome a separate user  story related to, for example, the need of a researcher working with data/information about or generated by Indigenous  Peoples to declare / describe adherence to the CARE-principles. A knowledge/information representation framework could support describing adherence to various data  governance principles in use around the world without implicitly or explicitly endorsing any of these principles.
